pkg/scheduler/factory.go

```diff
@@ -290,6 +291,12 @@ func (c *Configurator) createFromConfig(policy schedulerapi.Policy) (*Scheduler,
 	// Combine all framework configurations. If this results in any duplication, framework
 	// instantiation should fail.
 	var defaultPlugins schedulerapi.Plugins
+	// "PrioritySort" is neither a predicate nor priority before. We make it as the default QueueSort plugin.
+	defaultPlugins.Append(&schedulerapi.Plugins{
+		QueueSort: &schedulerapi.PluginSet{
+			Enabled: []schedulerapi.Plugin{{Name: queuesort.Name}},
+		},
+	})
 	defaultPlugins.Append(pluginsForPredicates)
 	defaultPlugins.Append(pluginsForPriorities)
 	defaultPlugins.Apply(c.plugins)
```

Comment:
should we disallow specifying both Plugins and Policy in validation? Since all predicates and priorities can now be expressed using plugins, I see no value in allowing to specify both. But that is probably for a separate PR.

Reply:
I agree. We can have a separate PR addressing this.

Reply:
were a PR or issue opened for this?